weblate/machinery/base.py

```diff
@@ -737,6 +738,8 @@ def get_glossary_id(
         if not glossary_tsv:
             return None
 
+        glossary_tsv = self.cleanup_glossary_tsv(glossary_tsv)
```

Comment:
What was your motivation for doing the cleanup as a separate step instead of doing that while generating it?

Reply:
Well, that was just to have the logic isolated to make it easier to test, but I can definitely just include that in the `get_glossary_tsv` method.